CMakeLists.txt

```diff
@@ -0,0 +1,592 @@
+CMAKE_MINIMUM_REQUIRED (VERSION 2.8)
+INCLUDE (CheckIncludeFiles)
+INCLUDE (CheckTypeSize)
+INCLUDE (CheckFunctionExists)
+INCLUDE (CheckSymbolExists)
+INCLUDE (CheckCSourceCompiles)
+INCLUDE (CheckStructHasMember)
+INCLUDE (CheckLibraryExists)
+
+PROJECT (cares C)
```

Comment:
One last nitpick. The project name should be ``c-ares``. And you should not use the project name to name the targets. Please use a plain ``add_library(cares ...`` there.